core/jazz_admin/README.md

```diff
@@ -0,0 +1,2 @@
+## Platform API for Retriving Installer JSON.
```

Comment:
Core API to perform administrative tasks

Reply:
That comment was for line 1. Can you update it?

Reply:
done